src/NLog/LogFactory.cs

```diff
@@ -338,7 +338,7 @@ public LoggingConfiguration Configuration
                         }
                     }
 
-                    this.OnConfigurationChanged(new LoggingConfigurationChangedEventArgs(value, oldConfig));
+                    this.OnConfigurationChanged(new LoggingConfigurationChangedEventArgs(oldConfig, value));
```

Comment:
No it is not a breaking change as I have "fixed" the now obsolete properties for LoggingConfigurationChangedEventArgs. Guess it should wait for NLog 4.5 because of the obsolete handling